go/interpreter/lfvm/instruction_test.go

```diff
@@ -13,15 +13,27 @@ package lfvm
 import "testing"
 
 func TestInstruction_String(t *testing.T) {
-	instruction := Instruction{opcode: STOP, arg: 0x0000}
-	if got, want := instruction.String(), "STOP"; got != want {
-		t.Errorf("Instruction.String() = %q, want %q", got, want)
+
+	tests := []struct {
+		instruction Instruction
+		want        string
+	}{
+		{Instruction{opcode: STOP, arg: 0x0000}, "STOP"},
```

Comment:
I thought we were going to start using stringer generated code to prevent this kind of testing, no?

Reply:
I guess, but there is no infrastructure so far, this is just migrating the opcodes.